examples/flows/evaluation/eval-qna-non-rag/f1_score.py

```diff
@@ -0,0 +1,54 @@
+from promptflow import tool
+from collections import Counter
+
+
+@tool
+def compute_f1_score(ground_truth: str, answer: str) -> str:
+    import string
+    import re
+
+    class QASplitTokenizer:
+        def __call__(self, line):
+            """Tokenizes an input line using split() on whitespace
+
+            :param line: a segment to tokenize
+            :return: the tokenized line
+            """
+
+            return line.split()
+
+    def normalize_text(text) -> str:
+        """Lower text and remove punctuation, articles and extra whitespace."""
+
+        def remove_articles(text):
+            return re.sub(r"\b(a|an|the)\b", " ", text)
+
+        def white_space_fix(text):
+            return " ".join(text.split())
+
+        def remove_punc(text):
```

Comment:
there are some unrecognized wordings, you can check the result from [Spell_Check result](https://github.com/microsoft/promptflow/actions/runs/7136348794/job/19434657799?pr=1412)

![image](https://github.com/microsoft/promptflow/assets/38847871/2d75ab3e-853b-4635-8612-fa78dd033da4)

Reply:
if these are expected wording, you can add them in [this file](https://github.com/microsoft/promptflow/blob/main/.cspell.json) inside your PR, then our spell check will recognize them and the check will pass.